items/weapons.md

```diff
@@ -0,0 +1,51 @@
+# Weapons
+Weapons are items that can be equipped
+
+## General Template
+As well as the base items [General Template](README.md#general-template), weapons define these:
+
+### Dice
+The dice that are rolled for an attack made with this. See [Attacks](../systems/rolls.md#attacks)
+
+### Damage Type
+The type of damage this weapon deals by default. See [Damage Types](../systems/combat.md#damage-types)
+
+### Traits
+A list of [weapon traits](#weapon-traits) this weapon has.
+
+### Range
+The maximum effective range on this weapon.
+
+# Melee weapons
+
+|Name           |Description                                                  |Weight |Price  |Dice   |Damage Type  |Traits                         |Range  |
+|---------------|-------------------------------------------------------------|-------|-------|-------|-------------|-------------------------------|-------|
+|Sword          |A sword                                                      |2      |25     |2d6    |Slashing     |1-handed                       |5ft    |
```

Comment:
In the attack systems I define an attack as two times the weapon dice. The weapon dice is simply d6 or d8.
If this is unclear, then perhaps the weapons system should be ammended.

Reply:
I thought it was like that, but also thought this might add clarity.